flow/scripts/global_route.tcl

```diff
@@ -58,3 +58,7 @@ check_antennas -report_file $env(REPORTS_DIR)/antenna.log -report_violating_nets
 # Write SDC to results with updated clock periods that are just failing.
 # Use make target update_sdc_clock to install the updated sdc.
 source [file join $env(SCRIPTS_DIR) "write_ref_sdc.tcl"]
+
+if {![info exists standalone] || $standalone} {
+  exit
+}
```

Comment:
OR is called with -exit in the makefile so there is no reason these commands are repeated in every ORFS script. I have to remove them every time I debug an issue.

Reply:
This is not always true.  'make all' invokes scripts/run_all.tcl which does "set standalone 0" and invokes each script in turn where they should not exit.  This is much closer to the single run flow you prefer.

Reply:
Currently "standalone" variable controls both loading files at the beginning of each script as well as the exit at the end. Should we use different variables? Then we can append the exit control variable to the files generated by make issue.

Reply:
@jjcherry56 what is your workflow in which you don't want to exit?  Most steps can't be re-run in the same session so I generally do exit.  I'm not clear there is one setting that satisfies all needs.

Reply:
If standalone is zero it does not exit. If standalone is 1 -exit is used when calling OR, so there is no point in having this code.

Reply:
@jjcherry56 you are correct, it can be removed.

Reply:
Fixed on b37d7b2